core/controllers/reader.py

```diff
@@ -456,7 +381,7 @@ class ClassifyHandler(base.BaseHandler):
     REQUIRE_PAYLOAD_CSRF_CHECK = False
 
     @require_playable
-    def post(self, exploration_id):
+    def post(self, unused_exploration_id):
```

Comment:
oh, it's actually because @require_playable needs that field. otherwise the code gets a bit ugly replicating the @require_playable. It's also possible this will be used for future classifier work. added note.